client/checkout/classic/upe.js

```diff
@@ -38,6 +38,11 @@ jQuery( function ( $ ) {
 			forceNetworkSavedCards: getConfig( 'forceNetworkSavedCards' ),
 			locale: getConfig( 'locale' ),
 			isUPEEnabled,
+			apiVersion:
+				paymentMethodsConfig.link !== undefined &&
+				paymentMethodsConfig.card !== undefined
```

Comment:
Also, do I get it right that paymentMethodsConfig takes advantage of capabilities (with server PR in mind which adds StripeLink capability)?

Reply:
Indeed, available payment methods are elaborated in `UPE_Payment_Gateway::get_payment_method_ids_enabled_at_checkout`

Reply:
I see, thanks!